### DISCUSSION TITLE
I hope that the packaging can fully comply with the archlinux specification

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [BlackArch/blackarch#4193] [General]
Not only is the question [here](https://github.com/BlackArch/blackarch-pkgbuilds/pull/20#issuecomment-2295042883) not answered, but there are still a lot of packages in the repository with incorrect version numbers that have not been fixed, and even the android ndk has been out of date for a long time without anyone noticing.

The main point is the first one. In the future, there may still be packages without version numbers released that lose the r prefix, causing them to conflict with aur or other repositories or even archlinux repositories (if entering the extra repository).

**Comment by wh1t3h4ts** [comment]
Please ensure all PKGBUILDs follow Arch’s versioning rules with explicit pkgver/pkgrel to prevent conflicts.

Also, avoid dropping revision prefixes unless conflicts are correctly set to avoid overlaps with AUR or official repositories.
